org.eclipse.xtext.util/src/org/eclipse/xtext/util/UriExtensions.java

```diff
@@ -34,7 +34,7 @@ public URI toUri(String stringUri) {
 	 */
 	public URI toEmfUri(java.net.URI netUri) {
 		String decoded = toDecodedString(netUri);
```

Comment:
Is the method name `toDecodedString` a shameless lie?

Reply:
No, it's not :)

Reply:
Agree :)